make/devices.cmake

```diff
@@ -16,8 +16,10 @@ function(DEFINE_ARCH)
   #    BITSTREAM_EXTENSION <ext>
   #    RR_PATCH_TOOL <path to rr_patch tool>
   #    RR_PATCH_CMD <command to run RR_PATCH_TOOL>
+  #    [NO_PLACE]
```

Comment:
No place? -- Should this be something like "no pins" or something? The pnr still goes through a placement phase right?